selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/WorkingWithTerminalTest.java

```diff
@@ -384,4 +381,10 @@ private void openMC(String currentLocation) {
     terminal.typeIntoActiveTerminal(Keys.ENTER.toString());
     terminal.waitTextInFirstTerminal("Modify time");
   }
+
+  protected void waitTextInFirstTerminal() {
```

Comment:
To make the test clear it would better to express what kind of text we are waiting, in the method name.
For example, it could be `checkNewFileAppearedInTheHomeDirecory()`.